crates/iota-indexer/reliability.md

```diff
@@ -0,0 +1,15 @@
+
+### Quantitative Requirements:
+
+| **Requirement**                 | **Metrics**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | **Thresholds**                                                                                                                         | Notes                         |
+| ------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------- |
+| **1. Checkpoint Sync Lag**      | `latest_fullnode_checkpoint_sequence_number - max_committed_checkpoint_sequence_number`                                                                                                                                                                                                                                                                                                                                                                                                                              | - **Genesis**: Lag can be > 0 not more than 35 minutes <br><br>- **Post Genesis**: Lag can be > 0 not more than 5 minutes              | Should be a Time series panel |
+| **2. Data Ingestion Latencies** | `checkpoint_db_commit_latency`<br> `checkpoint_db_commit_latency_transactions`<br> `checkpoint_db_commit_latency_tx_insertion_order`<br> `checkpoint_db_commit_latency_objects`<br> `checkpoint_db_commit_latency_objects_snapshot`<br> `checkpoint_db_commit_latency_objects_history`<br> `checkpoint_db_commit_latency_events`<br> `checkpoint_db_commit_latency_event_indices`<br> `checkpoint_db_commit_latency_packages`<br> `checkpoint_db_commit_latency_tx_indices`<br> `checkpoint_db_commit_latency_epoch` | - **Genesis**: ≤ 15 minutes.<br><br>- **Post Genesis**: ≤ 15 seconds.                                                                  |                               |
```

Comment:
Wouldn't it make sense to have a separate threshold for each latency?

What do these represent? The commit for a single transaction or a  batch of transactions? Is `15 sec` acceptable for all?

Reply:
fixed: https://github.com/iotaledger/iota/pull/6694/commits/8302fab3e7175af425c09d9564c90b04b405723f